docs/zrc/functionele_beschrijving.md

```diff
@@ -0,0 +1,15 @@
+
+#Functionele Documentatie ZAPI ZaakRegistratieComponent (ZRC)
+
+## Inhoud
+* [Introductie](#introductie)
+* [maakZaakAan](#maakZaakAan)
+
+## Introductie
+
+In dit document wordt de [API](https://github.com/VNG-Realisatie/gemma-zaken/blob/master/api-specificatie/zrc/openapi.yaml) voor de [ZaakRegistratieComponent] (https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-a97b6545-d5a7-485d-9b13-3ce22db5b9cf) functioneel beschreven.
+
+## maakZaakAan
```

Comment:
Er is geen maakZaakAan service ghehehe

Wellicht verschillen onze interpretaties van functionele omschrijving ook.

Ik zelf dacht meer aan: https://developers.google.com/gmail/api/guides/ maar nu ik er over na denk moeten er een aantal soorten documentatie komen. Moeten we wellicht even bespreken.

Reply:
Er moeten inderdaad meerdere soorten documentatie komen.

Deze documentatie was bedoeld voor de niet-ontwikkelaar. Dus de architect van de gemeente die moet kunnen snappen wat de API doet zonder een yaml te kunnen/hoeven lezen.
 Of een productmanager van een leverancier die moet kunnen beoordelen of een API van toepassing is op het product wat hij/zij managet etc.
De documentatie van gmail is nog veel te technisch voor deze doelgroep. 

De vraag is hoeveel functionele documentatie een ontwikkelaar nodig heeft voor een API.  Die heeft denk ik meer aan voorbeelden, use cases en/of uitgewerkte userstories.

Maar ik ben er ook nog niet uit wat wijsheid is dus bespreken, graag!